crates/packager/src/error.rs

```diff
@@ -170,8 +170,13 @@ pub enum Error {
         filename: String,
     },
     /// Missing notarize environment variables.
-    #[error("Could not find APPLE_ID & APPLE_PASSWORD or APPLE_API_KEY & APPLE_API_ISSUER & APPLE_API_KEY_PATH environment variables found")]
+    #[error("Could not find APPLE_ID & APPLE_PASSWORD & APPLE_TEAM_ID or APPLE_API_KEY & APPLE_API_ISSUER & APPLE_API_KEY_PATH environment variables found")]
     MissingNotarizeAuthVars,
+    /// Missing norarize APPLE_TEAM_ID environment variable.
+    #[error(
+        "The team ID is now required for notarization with app-specific password as authentication. Please set the `APPLE_TEAM_ID` environment variable. You can find the team ID in https://developer.apple.com/account#MembershipDetailsCard."
+      )]
+    MissingNotarizeAuthTeamId,
```

Comment:
No need for this error variant, we just used this on tauri to provide a good error message for this breaking change (since we're still on alpha for this project, it's ok to break it).